CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 Latest
 ------
-
+* Fixed `aiaprep` to return properly sized map.
```

Comment:
aiaprep is for derotating, recentering, and scaling to exactly 0.6 arcsec/pixel and is therefore not appropriate for submaps.